src/mono/wasm/runtime/workers/dotnet-crypto-worker.js

```diff
@@ -256,6 +256,24 @@ async function sign(type, key, data) {
     return Array.from(new Uint8Array(signResult));
 }
 
+async function derive_bits(password, salt, iterations, hashAlgorithm, lengthInBytes) {
```

Comment:
It feels like this function should either a) take PBKDF2 as a parameter, or b) include it in the name; since it's not applying a general/generic KDF, but a specific one.

Reply:
Hm.  Since the calling code and the acting code get delivered together I guess the signature is allowed to be modified as needed, so... I suppose it works.

Reply:
Yeah, if these APIs were public or somehow permanent, I would take a much different approach.